twitter/twitter_endpoint.bal

```diff
@@ -109,7 +109,7 @@ public type TwitterConfiguration record {
     http:ClientEndpointConfig clientConfig = {};
 };
 
-remote function Client.tweet(string status, string... args) returns Status|error {
+public remote function Client.tweet(string status, string... args) returns Status|error {
```

Comment:
Is there any reason to make the implementation of client actions `public`?

Reply:
Getting error as below with new version of ballerina

error:wso2/twitter:0.9.24::twitter_endpoint.bal:52:5: no implementation found for the function 'retweet' of non-abstract object 'wso2/twitter:0.9.24:Client'
error:wso2/twitter:0.9.24::twitter_endpoint.bal:175:1: interface and implementation of function 'retweet' of non-abstract object 'wso2/twitter:0.9.24:Client' should have same visibility

Reply:
Okay. We should match the access modifier of both signature and implementation with Ballerina v0.990.3.